pkg/test/runner/runner.go

```diff
@@ -34,6 +32,22 @@ type Runner struct {
 	cmd           string
 	t             *testing.T
 	initialCommit string
+	kptBin        string
+}
+
+func getKptBin() (string, error) {
```

Comment:
I think taking an explicit path for `kpt binary` will be better (we can read from an env var).

I have noticed, people sometime install `kpt` using OS package manager etc and sometime that is installed in preferred paths and can cause confusion which binary is being used.